src/java.base/share/classes/java/security/ProviderException.java

```diff
@@ -49,7 +49,7 @@ public ProviderException() {
     }
 
     /**
-     * Constructs a ProviderException with the specified detail
+     * Constructs a {@code ProviderException} with the specified detail
```

Comment:
I'm going to keep `{@code String} `since I don't want to go back and change all occurrences in all files. Documentation changes are too subjective and destroy the brain.